src/request.js

```diff
@@ -225,6 +225,23 @@ class Request {
       this.client.instance().numberInsight.get({level: 'standard', number: number}, this.response.insightStandard.bind(this.response));
     });
   }
+
+  // sending messages
+
+  sendSms(first, second, third, command) {
```

Comment:
Question: why this variable naming convention and not calling them `to`, `from` etc. ?